dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -656,7 +656,8 @@ boolean isUnusedClassSymbolReference(String sourceClassName, ClassSymbol classSy
             Instruction instruction = instructionHandle.getInstruction();
             if (instruction instanceof CPInstruction) {
```

Comment:
@netdpb FYI, BCEL has an abstract class "CPInstruction" that represents an instruction that takes a constant pool index, and we already had this function to determine whether a constant pool index is unused by JVM instructions. (Therefore this function does not list "safe" operations)

Reply:
That's fine, as long as it doesn't list "dangerous" operations either.